File: server/DAL/Entities/Enums.cs
namespace AdviceAssignement.DAL.Entities
{
    public class Enums
    {
        public enum ElevatorStatus
        {
            Idle,
            MovingUp,
            MovingDown,
            OpeningDoors,
            ClosingDoors
        }

        public enum ElevatorDirection
        {
            None,
            Up,
            Down
        }

        public enum DoorStatus
        {
            Closed,
            Open
        }
    }
}

File: client/src/enums.js
/**
 * @typedef {number} ElevatorStatusEnum
 * @enum {ElevatorStatusEnum}
 */
export const ElevatorStatus = {
  Idle: 0,
  MovingUp: 1,
  MovingDown: 2,
  OpeningDoors: 3,
  ClosingDoors: 4,
};

/**
 * @typedef {number} ElevatorDirectionEnum
 * @enum {ElevatorDirectionEnum}
 */
export const ElevatorDirection = {
  None: 0,
  Up: 1,
  Down: 2,
};

/**
 * @typedef {number} DoorStatusEnum
 * @enum {DoorStatusEnum}
 */
export const DoorStatus = {
  Closed: 0,
  Open: 1,
};
